config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	BUGSNAG_API_KEY: '72a0364cb361fc8f0fdc6cbf605f9963'
```

Comment:
Can you add a task on CI to replace this key and other places?
What will happen if we commit with `BUGSNAG_API_KEY: ''`?
Will it compile?
I'm thinking in DX here.
We shouldn't enforce anyone to have a Bugsnag profile in order to compile our app, but it would be better if we left that empty and replace on build time.

Reply:
I cannot test it because my bugsnag account trial has expired :(
When I change BUGSNAG_API_KEY to `''` then build doesn't fail but I'm not sure if whether it still sends requests to bugsnag.